stglib/aqd/aqdutils.py

```diff
@@ -859,16 +860,16 @@ def add_attributes(var, dsattrs):
             sn = dsattrs["VECSerialNumber"]
         var.attrs.update(
```

Comment:
I think you should comment out the var.attrs.update as well, so it's not making an empty function call.

Reply:
Will block comment out var.attrs.update call in aqdutils.ds_add_attrs.add_attributes def. Also will go back and redo do black formatting on all altered code in this PR.